### DISCUSSION TITLE
How to add key/value as Request header of OIDCProviderTokenEndpoint

### DISCUSSION THREAD

**Discussion by RamyaAyyanan** [OpenIDC/mod_auth_openidc#863] [Q&A]
Hi ,
i need to pass API key& value in request header for OIDCProviderTokenEndpoint . i see , we can pass params as  Querystring for OIDCProviderTokenEndpointParams. is that possible to pass the key/value in header instead of Query string?

Thanks,
Ramya.A

**Comment by zandbelt** [comment]
no, that is not possible; it is also a non-standard feature that any standards compliant software should not use
